src/apps/concepts/components/EnhancedTableToolbar.tsx

```diff
@@ -70,7 +70,7 @@ export const EnhancedTableToolbar = (props: EnhancedTableToolbarProps) => {
       )}
       {numSelected > 0 ? (
         <>
-          {!showAddConcepts ? null : (
+          {showAddConcepts ? null : (
```

Comment:
I see what you're doing here, but this should actually still be:

```ts
{!showAddConcepts ? null : (
```

Because it's still possible to be viewing concepts without a dictionary to add them to